Code/Framework/AzToolsFramework/AzToolsFramework/ActionManager/Menu/MenuManagerInterface.h

```diff
@@ -35,23 +36,56 @@ namespace AzToolsFramework
         AZ_RTTI(MenuManagerInterface, "{D70B7989-62BD-447E-ADF6-0971EC4B7DEE}");
 
         //! Register a new Menu to the Menu Manager.
-        virtual MenuManagerOperationResult RegisterMenu(const AZStd::string& identifier,
-            const MenuProperties& properties) = 0;
+        //! @param menuIdentifier The identifier for the menu that is being registered.
+        //! @param properties The properties object for the newly registered menu.
+        //! @return An void outcome if successful, or a string with a message detailing the error otherwise.
+        virtual MenuManagerOperationResult RegisterMenu(const AZStd::string& menuIdentifier, const MenuProperties& properties) = 0;
 
-        //! Bind an action to a menu.
+        //! Register a new Menu Bar to the Menu Manager.
+        //! @param menuBarIdentifier The identifier for the menu bar that is being registered.
+        //! @return An void outcome if successful, or a string with a message detailing the error otherwise.
```

Comment:
nit: An void - A void (repeated below) - also maybe this should be something other than 'void', as this really is a success right? Not the absence of a result/return, so maybe `an empty AZ::Outcome` (might be clearer?)